src/main/java/com/github/psambit9791/jdsp/misc/Export.java

```diff
@@ -0,0 +1,246 @@
+package com.github.psambit9791.jdsp.misc;
+
+import org.apache.commons.math3.complex.Complex;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Locale;
+import java.util.Objects;
+
+/**
+ * <h1>Export</h1>
+ * The Export class provides methods for exporting data
+ * <p>
+ */
+public class Export {
```

Comment:
Sorry, wasn't aware of that, fixed it in the last commits.

I would however like to see the CSV-class more elaborate. The most used case of export to CSV would be if a user has done an operation on some data and receives a 1D or 2D double/Complex result, and wants to export that result. It would be easier if a user could just input a double[] or double[][] (or Complex[] and Complex[][]) into an export to CSV function.